apps/documentation/src/app/examples/combobox/combobox.tailwind.example.ts

```diff
@@ -11,7 +11,7 @@ import {
 } from 'ng-primitives/combobox';
 
 @Component({
-  selector: 'app-combobox-tailwind',
+  selector: 'app-combobox',
```

Comment:
We discovered as part of #314 that we need to keep `-tailwind` in the selector (class name is fine as is, just selector), otherwise angular will complain about selector collision, so we should keep this, and I guess if any other tailwind examples don't have it we should re-add it.